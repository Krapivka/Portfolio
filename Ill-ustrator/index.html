<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<link rel="stylesheet" type="text/css" href="assets/css/style.css">
	<link href="https://fonts.googleapis.com/css?family=Sintony&display=swap" rel="stylesheet">
	<title>Illustrator</title>
</head>
<body>
	<header class="header clearfix">
		<div class="container">
			<div class="header__main">
			<img src="assets/images/logo.png" class="logo">
			<img src="assets/images/ill-ustrator.png" class="ill">
			<p class="header__text">As if I was never going to make a free PSD withouta 
illustration of me in it?!</p>
			</div>
			<img src="assets/images/bio-bio.png" class="bio_bio">
		</div>	
	</header>
	<section>
		<div class="container">
			<div class="sect">
				<h2>illustrative works</h2>
				<p class="desc">Sed suscipit risus vel sapien fringilla elementum. Donec vulputate sapien suscipit nulla tincidunt sodales. Ut ut nisi in nisl convallis ultricies. Curabitur tristique faucibus tincidunt. Donec rutrum mi faucibus nisi luctus sodales facilisis neque semper.</p>
				<div class="blocks">
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				</div>
				<div class="mininav">
					<img class="mininav__el" src="assets/images/nav2.png">
					<img class="mininav__el"src="assets/images/nav2.png">
					<img class="mininav__el mininav__point"src="assets/images/nav1.png">
					<img class="mininav__el"src="assets/images/nav2.png">
				</div>
			</div>
		</div>	
	</section>
	<section>
		<div class="container">
			<div class="sect">
				<h2>Graphical work</h2>
				<p class="desc">Sed suscipit risus vel sapien fringilla elementum. Donec vulputate sapien suscipit nulla tincidunt sodales. Ut ut nisi in nisl convallis ultricies. Curabitur tristique faucibus tincidunt. Donec rutrum mi faucibus nisi luctus sodales facilisis neque semper.</p>
				<div class="blocks">
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				</div>
				<div class="mininav">
					<img class="mininav__el" src="assets/images/nav2.png">
					<img class="mininav__el"src="assets/images/nav2.png">
					<img class="mininav__el mininav__point"src="assets/images/nav1.png">
					<img class="mininav__el"src="assets/images/nav2.png">
				</div>
			</div>
		</div>	
	</section>
	<section>
		<div class="container">
			<div class="sect">
				<h2>corporates</h2>
				<p class="desc">Sed suscipit risus vel sapien fringilla elementum. Donec vulputate sapien suscipit nulla tincidunt sodales. Ut ut nisi in nisl convallis ultricies. Curabitur tristique faucibus tincidunt. Donec rutrum mi faucibus nisi luctus sodales facilisis neque semper.</p>
				<div class="blocks">
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				<div class="block__el"></div>
				</div>
				<div class="mininav">
					<img class="mininav__el" src="assets/images/nav2.png">
					<img class="mininav__el"src="assets/images/nav2.png">
					<img class="mininav__el mininav__point"src="assets/images/nav1.png">
					<img class="mininav__el"src="assets/images/nav2.png">
				</div>
			</div>
		</div>	
	</section>
	<section>
		<div class="container">
			<div class="sect clearfix sect__last">
				<div class="sect__gr">
				<h2>Design & illustrative</h2>
				<p class="desc">Sed suscipit risus vel sapien fringilla elementum. Donec vulputate sapien suscipit nulla tincidunt sodales. Ut ut nisi in nisl convallis ultricies. Curabitur tristique faucibus tincidunt. Donec rutrum mi faucibus nisi luctus sodales facilisis neque semper. <br><br>
				Curabitur euismod dignissim orci, at vestibulum ligula convallis non. Etiam sagittis diam quis felis volutpat convallis. In vehicula, est sit amet iaculis venenatis, mauris felis euismod tellus, sed facilisis eros felis in nunc. Nam nec arcu vel magna posuere malesuada. Fusce pretium magna vitae elit imperdiet dignissim. Duis egestas quam sit amet tortor vestibulum ac rhoncus dolor vulputate. Maecenas sit amet neque in est iaculis vestibulum. 
				</p>
				</div>
				<div class="sect__gr">
				<h2>Design & illustrative</h2>
				<p class="desc">Sed suscipit risus vel sapien fringilla elementum. Donec vulputate sapien suscipit nulla tincidunt sodales. Ut ut nisi in nisl convallis ultricies. Curabitur tristique faucibus tincidunt. Donec rutrum mi faucibus nisi luctus sodales facilisis neque semper.<br><br>
				Curabitur euismod dignissim orci, at vestibulum ligula convallis non. Etiam sagittis diam quis felis volutpat convallis. In vehicula, est sit amet iaculis venenatis, mauris felis euismod tellus, sed facilisis eros felis in nunc. Nam nec arcu vel magna posuere malesuada. Fusce pretium magna vitae elit imperdiet dignissim. Duis egestas quam sit amet tortor vestibulum ac rhoncus dolor vulputate. Maecenas sit amet neque in est iaculis vestibulum. 
				</p>
				<div>
			</div>
		</div>	
	</section>
	<footer class="clearfix">
		<div class="container">
			<div class="footcont">
				<img src="assets/images/logo.png" class="logo">
				<p class="foot__text">It'll be ute when mad as a waratah. You little ripper rort where as busy as a captain </p>
				<div class="block__black clearfix">
					<p class="block__text">Join our email newsletter</p>
					<img src="assets/images/email.png" class="mail">
				</div>
			</div>
			<div class="social">
				<div class="social__el fac"><img src="assets/images/icon4.png" class="icon"></div>
				<div class="social__el twit"><img src="assets/images/icon3.png" class="icon"></div>
				<div class="social__el ball"><img src="assets/images/icon2.png" class="icon"></div>
				<div class="social__el wifi"><img src="assets/images/icon1.png" class="icon"></div>
			</div>
		</div>	
	</footer>
</body>
</html>