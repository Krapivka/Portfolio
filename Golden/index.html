<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<link rel="stylesheet" type="text/css" href="assets/css/style.css">
	<title>
		Golden
	</title>
</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img class="logo" src="assets/images/logo.png">
				<nav>
					<ul class="menu">
						<li>
							<a href="#">Home</a>
						</li>
						<li>
							<a href="#">Services</a>
						</li>
						<li>
							<a href="#">Portfolio</a>
						</li>
						<li>
							<a href="#">About</a>
						</li>
						<li>
							<a href="#">Contact</a>
						</li>
					</ul>
				</nav>
			</div>
			<div class="titles">
				<div class="titles__first">
					Welcome To Our Studio! 
				</div>
				<h1>
					It's nice to Meet you
				</h1>
			</div>
			<a class="button" href="">TELL ME MORE</a>
		</div>
	</header>
	<section>
		<div class="container">
			<div class="title">
				<h2>
					Services
				</h2>
				<p>
					Proin iaculis purus consequat sem cure. 
				</p>
			</div>
			<div class="rewards clearfix">
				<div class="rewards__el">
					<img class="rewards__logo" src="assets/images/icon1.png">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="rewards__el">
					<img class="rewards__logo" src="assets/images/icon2.png">
					<h3>Responsive Web</h3>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="rewards__el__last ">
					<img class="rewards__logo" src="assets/images/icon3.png">
					<h3>Web Security</h3>
					<p>Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>
		</div>
	</section>
	<section>
		<div class="sec__second">
			<div class="container">
				<div class="title">
					<h2>
						Our portfolio
					</h2>
					<p>
						Proin iaculis purus consequat sem cure. 
					</p>
				</div>
				<div class="img clearfix">
					<img class="img__el" src="assets/images/img1.jpg">
					<img class="img__el" src="assets/images/img2.jpg">
					<img class="img__el__last" src="assets/images/img3.jpg">
				</div>
			</div>
		</div>
	</section>
	<section>
		<div class="container">
			<div class="title">
					<h2>
						About Us
					</h2>
					<p>
						Proin iaculis purus consequat sem cure.
					</p>
				</div>
			<div class="about clearfix">
				<div class="about__el__left clearfix">
					<div class="about__content__left">
					<h4>
									July 2010<br>
						Our Humble Beginnings
					</h4>
					<p class="about__me__left">
						Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.
					</p>
					</div>
					<img class="about__img__left" src="assets/images/about1.jpg">
				</div>
				<div class="about__el__right clearfix">
					<div class="about__content__right">
					<h4>
									January 2011<br>
							Facing Startup Battles
					</h4>
					<p class="about__me__right">
						Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. 
					</p>
					</div>
					<img class="about__img__right " src="assets/images/about2.jpg">
				</div>
				<div class="about__el__left clearfix">
					<div class="about__content__left">
					<h4>
									December 2012<br>
Enter The Dark Days
					</h4>
					<p class="about__me__left">
						Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.
					</p>
					</div>
					<img class="about__img__left" src="assets/images/about3.jpg">
				</div>
				<div class="about__el__right clearfix">
					<div class="about__content__right">
					<h4>
									February 2014<br>
										Our Triumph
</h4>
					<p class="about__me__right">
						Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. D
					</p>
					</div>
					<img class="about__img__right" src="assets/images/about4.jpg">
				</div>
				
			</div>
		</div>
	</section>
	<footer>
		<div class="container">
			<p class="copyright"> Â© Copyright 2014 FreebiesXpress.com</p>
		</div>
	</footer>
</body>
</html>